src/main/java/edu/wpi/grip/ui/SocketControlView.java

```diff
@@ -38,17 +45,17 @@
  * between EventBus events and a JavaFX property.
  */
 public class SocketControlView extends GridPane implements Initializable {
+
     @FXML
     private Label identifier;
 
     @FXML
     private StackPane controlPane;
 
-    // The "handle" is a simple radio button next ot the socket identifier that shows weather or not there is a
-    // connection going to or from the socket.  If there is such a connection, the ConnectionView is rendered as a
-    // curve going from one handle to another.
-    @FXML
-    private RadioButton handle;
+    // The "handle" is a simple shape next ot the socket identifier that shows weather or not there is a connection
+    // going to or from the socket.  If there is such a connection, the ConnectionView is rendered as a curve going
+    // from one handle to another.
```

Comment:
Turn this into a javadoc comment. You are describing the object and its purposes.